src/main/java/core/basesyntax/controller/ConsoleHandler.java

```diff
@@ -0,0 +1,74 @@
+package core.basesyntax.controller;
+
+import core.basesyntax.dao.BetDao;
+import core.basesyntax.dao.BetDaoImpl;
+import core.basesyntax.dao.UserDao;
+import core.basesyntax.dao.UserDaoImpl;
+import core.basesyntax.model.Bet;
+import core.basesyntax.model.User;
+import java.util.Scanner;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+public class ConsoleHandler {
+    private static final String NAME_REGEX = "[a-zA-Z]+";
+    private static final String EMAIL_REGEX = "[a-zA-Z0-9._%+-]+@[a-zA-Z0-9.-]+\\.[a-zA-Z]{2,6}$";
+    private static final String WRONG_DATA_INSERTION = "Будь ласка, введіть коректні дані";
+    private static final String WHITESPACE_SEPARATOR = " ";
+    private static final String QUIT_SIGH = "q";
+    private BetDao betDao = new BetDaoImpl();
+    private UserDao<User> users = new UserDaoImpl();
+
+    private Scanner scanner = new Scanner(System.in);
+
+    public void handle() {
+        while (true) {
+            String command = scanner.nextLine();
+            if (command.equalsIgnoreCase(QUIT_SIGH)) {
+                return;
+            }
+            Bet bet = null;
+            try {
+                String[] betData = command.split(WHITESPACE_SEPARATOR);
+                int value = Integer.parseInt(betData[0]);
+                double risk = Double.parseDouble(betData[1]);
+                bet = new Bet(value, risk);
+            } catch (ArrayIndexOutOfBoundsException | NumberFormatException e) {
+                System.out.println(WRONG_DATA_INSERTION);
+            }
+            if (bet != null) {
+                betDao.add(bet);
+                System.out.println(bet);
+
+            }
+        }
+    }
+
+    public void handleUser() {
+        while (true) {
+            String command = scanner.nextLine();
+            if (command.equalsIgnoreCase(QUIT_SIGH)) {
+                return;
+            }
+            User user = null;
+            Pattern patternName = Pattern.compile(NAME_REGEX);
+            Pattern patternEmail = Pattern.compile(EMAIL_REGEX);
+            try {
+                String[] betData = command.split(WHITESPACE_SEPARATOR);
+                Matcher matcherName = patternName.matcher(betData[0]);
+                Matcher matcherEmail = patternEmail.matcher(betData[1]);
+                if (!matcherName.matches() || !matcherEmail.matches()) {
+                    throw new NumberFormatException();
```

Comment:
when you throw exception it's better to add some message to it 
Why do we throw NumberFormatException if it's email and name validation, throw RuntimeException in this case

Reply:
I made it by this way, because i have catch block with this type of exception (NumberFormatException) , so catch block correctly deal with this type of exception (my catch has 2 exception , so appearence of third exception lead to creation additional catch block). If i change NumberFormatException by RuntimeException i have to add additional catch block , which made my code bigger without additional functionality. About message in catch block: i have appropriate message in catch block, which inform about wrong data input.